packages/react-server-dom-webpack/src/shared/ReactFlightClientReference.js

```diff
@@ -0,0 +1,45 @@
+/**
```

Comment:
This file should maybe be called `ClientReferenceMetaData` or maybe we go away from the ClientReference naming all together and call it `ImportMetaData` or something.

Because there is a notion of a ClientReference and ServerReference - the reified thing (e.g. proxy or real thing) - which is not this. And there's no code in this file related to it, and there shouldn't be because that code can't be shared before server/client.

So this file shouldn't be called `ReactFlightClientReference.js`.

Reply:
yeah this makes sense to me